tests/phpunit/tests/functions/isWpVersionCompatible.php

```diff
@@ -85,6 +105,34 @@ public function data_is_wp_version_compatible() {
 				'required' => array(),
 				'expected' => true,
 			),
+
+			// Improper WP version.
+			'improper trailing x.x.0'   => array(
+				'required' => '5.2.0',
+				'expected' => true,
+				true,
+			),
+			'correct version'           => array(
+				'required' => '5.2',
+				'expected' => true,
+			),
+			'incorrect trailing x.0.0'  => array(
+				'required' => '5.0.0',
+				'expected' => true,
+				true,
+			),
+			'correct version x.0'       => array(
+				'required' => '5.0',
+				'expected' => true,
+			),
+			'correct version x.0.1'     => array(
+				'required' => '5.0.1',
+				'expected' => true,
+			),
```

Comment:
Is it worth adding a test to ensure the version is converted correctly? It might require some code shuffling.

Reply:
Provided there are tests covering all the bases, testing the output with each input should be enough, and the respective lines can be checked for coverage in a coverage report.

Reply:
I added this test to ensure a `.0` in the middle of a version number wouldn’t be changed.

Reply:
I will add some new tests for that. I'll have it fixed by tomorrow.